cpu/native/Makefile.features

```diff
@@ -12,6 +12,7 @@ endif
 FEATURES_PROVIDED += periph_cpuid
 FEATURES_PROVIDED += periph_eeprom
 FEATURES_PROVIDED += periph_flashpage
+FEATURES_PROVIDED += periph_flashpage_in_address_space
```

Comment:
If  a new cpu feature is provided it needs to be provided for that cpu not just one sys-module

Reply:
I dont understand what you mean ? it is provided for the cpu which is native is this case

Reply:
Fixed in an Murdock is green :) Will squash when you say looks fine